Makefile

```diff
@@ -0,0 +1,13 @@
+# Makefile for Truss.
+#
+# Build native Truss by default.
+default:
+	$(MAKE) -C truss
+
+# Run the go tests and the truss integration tests
+test:
+	go test -v ./...
```

Comment:
Initially I balked at the idea of running tests on vendored code, but after thinking about it for a second, the concept of "vendoring" is that it becomes *your* code. So it makes sense to test it. Now, what is the likelihood of the vendored dependencies being broken? Pretty low, admittedly, for any dependency that's worth a hoot. However, the goal of software is--expressed in one manner--to eliminate uncertainty. In this regard, testing the code that we have adopted as our own seems reasonable.

We can always create an alternate make target (e.g., `test-nv`) to exclude vendored tests to help speed things up during development cycles, if that's the objection. Unit tests, properly categorized as unit tests, should execute very quickly though.